### PR TITLE
Add GDG Devfest Hamburg 2017.

### PR DESCRIPTION
**Pull request by serj-lotutovici**
* Note that only the CFP is up to date. The website will be updated shortly.

### CODE REVIEW

File: _conferences/devfest-hamburg-2017.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: "GDG Devfest"
+website: https://hamburg.devfest.de/
+location: Hamburg, Germany
+
+date_start: 2017-11-25
+date_end:   2017-11-26
+
+cfp_start: 2017-07-26
+cfp_end:   2016-10-01
```
Comment (NightlyNexus) [inline _conferences/devfest-hamburg-2017.md]: 2017?

Comment (serj-lotutovici) [reply _conferences/devfest-hamburg-2017.md]: ouch... thx! fixed!

**Comment by JakeWharton** [comment]
I don't think we should delete the old one. There's potential for future data endeavors with them.

**Comment by serj-lotutovici** [comment]
@JakeWharton restored the deleted file.

**Pull request merged**